tools/ci.sh

```diff
@@ -116,6 +116,7 @@ function mainTask()
   if [ "${DEV_BUILD_DISABLED:-false}" = "true" ]; then
     echo "$(date +%FT%T%z) [CI] Dev build disabled. Skipping..."
   else
+    yarn install
```

Comment:
@zondervancalvez The configure script already runs `yarn install`, must we run it twice, e.g. during and right before the configure script as well?

Reply:
We updated the `yarn run configure` script so that it will run first the `yarn install` command.